src/graphql/Hooks.graphql

```diff
@@ -69,6 +69,11 @@ type Hook {
   status(hookGroupId: ID = hookGroupId, hookId: ID = hookId): HookStatus
 }
 
+type HookGroup {
+  hookGroupId: ID!
+  hooks(hookGroupId: ID = hookGroupId): [Hook]
```

Comment:
Let's put a blank line here just so we can have a clear division between properties and resolved methods.